gtsfm/loader/hilti_loader.py

```diff
@@ -120,12 +235,33 @@ def __read_lidar_pose_priors(self) -> Dict[int, Pose3]:
 
         return w_T_imu
 
+    def __read_fastlio_poses(self) -> List[Pose3]:
+        """Read the poses for the IMU for rig indices."""
+        filepath = str(self._base_folder / GT_POSE_RELATIVE_PATH)
```

Comment:
does this work? thought you needed self._base_folder + "/" + GT_POSE_RELATIVE_PATH

Reply:
Nope thats the benefit of creating `Path` objects instead of str.

Reply:
oh I see its a path object..